src/components/UserProfile/UserProfileContent.js

```diff
@@ -1,6 +1,7 @@
-import React from "react";
+import React from 'react'
 import styled from 'styled-components/macro'
-import { Link } from "react-router-dom";
+import { Link } from 'react-router-dom'
+import ShareUserProfile from '../ShareProfile/ShareUserProfile'
 
 export default function UserProfileContent({ userDataInfo }) {
```

Comment:
Think ybout getting rid of the UserProfile namespacing, it is in a fodler which is already named UserProfile.